ES/ESmy019.xml

```diff
@@ -31,63 +30,154 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?><TEI
+                           </binding>
+                     </bindingDesc>
+                     </physDesc>
+
+                     <history>
+                        <origin>
+                           <origDate notBefore="1841" notAfter="1867">1841-67</origDate> for
+                           <ref target="#p1"/>, and <origDate notBefore="1820" notAfter="1867"> the same or
+                         slightly earlier time </origDate> for <ref target="#p2"/>.
+                         At an unknown time, <ref target="#p2">Unit 2</ref> was included into the binding of
+                        <ref target="#p1">Unit 1</ref>.
+                        </origin>
+                     </history>
+
+                     <msPart xml:id="p1">
+                        <msIdentifier/>
+                        <msContents>
+                           <summary/>
+
+                           <msItem xml:id="p1_i1">
+                              <locus from="2ra" to="17va"/>
+                              <title type="complete" ref="LIT6166MiraclesJohnBaptist">Ten
+                                miracles of John the Baptist</title>
+                              <incipit xml:lang="gez"/>
+                              <note>
+                                  Cp. Mss Gunda Gunde GG-024, RQM-029.
+                              </note>
+                              </msItem>
+
+                              <msItem xml:id="p1_i2">
+                                 <locus from="17vb" to="19rb"></locus>
+                                 <title type="complete" ref="LIT4717TaammerLi">Two miracles of Libānos</title>
+                                 <incipit xml:lang="gez"/>
+                                 <explicit xml:lang="gez"></explicit>
+                                 <note>
+                                   For the edition, s. <bibl><ptr target="bm:Bausi2003Libanos"/>
+                                   <citedRange unit="paragraph">282-283</citedRange></bibl>, and
+                                   <bibl><ptr target="bm:Bausi2003Libanos"/>
+                                   <citedRange unit="paragraph">290-291</citedRange></bibl>.
+                                   The red ink of  is faded and poorly readable.
+                                   The subscription (<locus from="18vb" to="19ra"/>) states
+                                   that the manuscript was purchased and donated to the church of Dabra
+                                   Maʿṣo by <persName ref="PRS13128GadlaGiyorgis">Gadla Giyorgis</persName> and
+                                   <persName role="donor" ref="PRS13127TasfaMaryam">Tasfā Māryām</persName>. “Condemnatio” mentions
+                                   <persName ref="PRS8388SalamaI">ʾabuna Salāmā III</persName> (in tenure <date>1841-67</date>).
+                                    </note>
+                                 </msItem>
+                                 <msItem xml:id="p1_i3">
```

Comment:
is it sure a msItem and not guest text? it comes after a donation note, looks like an addition to me

Reply:
Before it was an addition, indeed, but it is finely written and it it is an independent  record now. It must not be always an addition, it is rather a boundary case. We highlight a scribal hand which is not that of a student (now I do not remember if it attested elsewhere in MY mss). BUt if you insist, i can scale it down to the  addition